tools/cbpManager.deploy/shinyproxy/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM openjdk:8-jre
+
+RUN mkdir -p /opt/shinyproxy/
+RUN wget https://www.shinyproxy.io/downloads/shinyproxy-2.5.0.jar -O /opt/shinyproxy/shinyproxy.jar
+COPY application.yml /opt/shinyproxy/application.yml
+
+WORKDIR /opt/shinyproxy
+CMD ["java", "-jar", "/opt/shinyproxy/shinyproxy.jar"]
```

Comment:
Is there anything wrong with the official shinyproxy docker image?
Otherwise 
```
FROM openanalytics/shinyproxy:2.5.0
COPY application.yml /opt/shinyproxy/application.yml
```
could already be sufficient.

Reply:
Yes, I tried with openanalytics/shinyproxy but got the error that the .jar file could not be found. This solution worked for me.

Reply:
I had issues with starting the nginx container. Very strange. 
In that case I would recommend to use a minimalistic JRE container like it's already in use with FhirSpark:
```FROM gcr.io/distroless/java-debian10:11``` 
It's that much minimalistic that you will need to download the jar file in a separate builder container.

Reply:
After `docker-compose build` and `docker-compose up -d` the nginx container is not running permanently and is stopped, this behaviour is normal. If you successfully log into ShinyProxy the nginx container starts.

Reply:
I was able to login, but the the application did not load and threw an error 500 "Failed to start container".